src/main/java/com/exasol/adapter/jdbc/BaseTableMetadataReader.java

```diff
@@ -84,12 +107,12 @@ protected boolean isTableSelected(final String tableName, final List<String> sel
         return (selectedTables == null) || selectedTables.isEmpty() || selectedTables.contains(tableName);
     }
 
-    protected void mapOrSkipSelectedTable(final ResultSet remoteTables, final List<TableMetadata> translatedTables,
-            final String tableName) throws SQLException {
+    protected boolean tableIsNotInExclusionsList(final String tableName) {
```

Comment:
Use positive formulation. Negation in names is confusing. Double negation more so!